src/Tribe/PUE/Checker.php

```diff
@@ -1003,6 +1003,9 @@ public function ajax_validate_key() {
 		 * @return string
 		 */
 		private function get_api_message( $info ) {
+			// Make sure Thickbox is available and consistent appearance regardless of which admin page we're on
+			add_thickbox();
```

Comment:
This seems like the wrong place to do this. Where we use the message to add a notice, we should do that there.

Reply:
Yeah... `ajax_validate_key()` is the only place I traced this usage back to, but enqueuing within an ajax response probably isn't right either... so I just undid this.